changelog/unreleased/3380

```diff
@@ -0,0 +1,8 @@
+Bugfix: ANR after removing account with too many downloaded files
+
+Previously, when a user account was deleted, the application would freeze when trying to delete a large number of files.
+Now, after changing a method, we have managed to prevent the application from freezing and working correctly.
```

Comment:
I would say it this way, as it didn't always happen. Also, I think it's better to use passive voice in this short changelog messages, but that's my opinion 😃 :
```suggestion
Previously, when a user account was deleted, the application could freeze when trying to delete a large number of files.
Now, the application has been fixed so that it doesn't freeze anymore by doing this.
```

Reply:
Thanks dude :)